index.js

```diff
@@ -0,0 +1,27 @@
+var express = require('express');
```

Comment:
Unnecessary file?

I would create the app here, the correct one. Then in the `api/index.js` I would add the routes that are relative to the api (including graphql).